deatonjm-austin

```diff
@@ -0,0 +1,6 @@
+# Austin
+
+### Things to do
+- Alamo Drafthouse
+- Pinballz
```

Comment:
@githubteacher You can! Tons of old school arcade and pinball machines and it's BYOB